common/templatetags/common_tags.py

```diff
@@ -26,6 +29,34 @@ def richtext_inline(value):
     ))
 
 
+@register.filter
+def richtext_aside(value):
+    "Returns HTML-formatted rich text with span injected in each block level element"
+    html_str = richtext(value).__html__()
+
+    # Implicit cache invalidation happens based on the string
+    cache_key = f"aside_html_cache_{hashlib.md5(html_str.encode('UTF-8')).hexdigest()}"  # nosec
```

Comment:
This is kind of a minor issue but I do think using `nosec` to ignore _all_ security alerts is possibly a bit too broad. I'd feel more comfortable with either ignoring the specific alert we're encountering `nosec B324` or using the hashlib argument `usedforsecurity=False`, as we have done elsewhere: 
https://github.com/freedomofpress/pressfreedomtracker.us/blob/57a8ea99a23f09cbd84ffb5b315064da1390eeb2/common/utils/mailchimp.py#L68-L71

Reply:
I like the `usedforsecurity=False`. Makes sense to me.